frontend/src/AppRoutes/pageComponents.ts

```diff
@@ -15,6 +15,10 @@ export const ServiceMapPage = Loadable(
 	() => import(/* webpackChunkName: "ServiceMapPage" */ 'modules/Servicemap'),
 );
 
+export const SaveViews = Loadable(
+	() => import(/* webpackChunkName: "SaveViews" */ 'pages/LogsModulePage'),
```

Comment:
Isn't this common to Logs and Traces?

Reply:
We have to create a wrapper around it and will do it while working on the traces view. Added comment to keep a track.